Angular/angular-mentoring/src/app/course-item/course-item.component.ts

```diff
@@ -20,10 +23,17 @@ export class CourseItemComponent implements OnInit {
   @Output('onDeleteCourse') onDelete: EventEmitter<number> = new EventEmitter<number>();
 
   ngOnInit(): void {
+    const isInFuture = this.item.creationDate > new Date();
+    const isNewerThanAFortnite = (new Date().getTime() - this.item.creationDate.getTime()) < 86400000 * 14;
```

Comment:
We could make this logic more readable by introducing constants for the “magic numbers” 86400000 and 14.

Actually, it would make more sense to put this time calculation into a function.

This is optional, but consider doing it :)